middleware/admin/verify_attestation.py

```diff
@@ -122,7 +142,9 @@ def do_verify_attestation(options):
     ui_message = bytes.fromhex(ui_result[1])
     ui_hash = bytes.fromhex(ui_result[2])
     mh_len = len(UI_MESSAGE_HEADER)
-    if ui_message[:mh_len] != UI_MESSAGE_HEADER:
+    try:
+        validate_ui_message_header(ui_message)
+    except Exception:
```

Comment:
Do we need the validation functions to raise an error? Can't we just raise one if the validation function returns `False`?

Reply:
Sounds good, I'll have these validation functions return a bool instead